.ci/docker-compose-file/python/pytest.sh

```diff
@@ -1,21 +1,29 @@
 #!/bin/sh
 
 ## This script is to run emqx cluster smoke tests (fvt) in github action
-## This script is executed in pacho_client
+## This script is executed in paho_client
 
 set -x
 set +e
 
-LB="haproxy"
+EMQX_TEST_DB_BACKEND=$1
+if [ "$EMQX_TEST_DB_BACKEND" = "rlog" ]
+then
+  # target only replica to avoid replication races
```

Comment:
Please mark it as a TODO and refer an issue

Reply:
I created this issue: https://github.com/emqx/emqx/issues/6094